quartz/plugins/emitters/aliases.ts

```diff
@@ -14,20 +13,7 @@ export const AliasRedirects: QuartzEmitterPlugin = () => ({
 
     const { argv } = ctx
     for (const [_tree, file] of content) {
-      const dir = path.posix.relative(argv.directory, path.dirname(file.data.filePath!))
-      const aliases = file.data.frontmatter?.aliases ?? []
-      const slugs = aliases.map((alias) => path.posix.join(dir, alias) as FullSlug)
-      const permalink = file.data.frontmatter?.permalink
-      if (typeof permalink === "string") {
-        slugs.push(permalink as FullSlug)
-      }
-
-      for (let slug of slugs) {
-        // fix any slugs that have trailing slash
-        if (slug.endsWith("/")) {
-          slug = joinSegments(slug, "index") as FullSlug
-        }
-
```

Comment:
revert this for graph fastRebuild, or use the functions you wrote in frontmatter

Reply:
Oh, file.data.aliases is not populated here, isn't it
fixed